.github/kokoro/database.cfg

```diff
@@ -0,0 +1,14 @@
+build_file: "symbiflow-prjxray-db/.github/kokoro/database.sh"
```

Comment:
This file shouldn't exist for symbiflow-arch-defs (bad copy from prjxray).

Reply:
Removed